examples/python/template/main.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+"""
+Example template.
+
+Run:
+```sh
+pip install -r examples/python/template/requirements.txt
+./examples/python/template/main.py
+```
+"""
+from __future__ import annotations
+
+import argparse
+import logging
+
+import rerun as rr  # pip install rerun-sdk
+
+
+def setup_logging() -> None:
+    # Forward all text logs to Rerun by attaching a handler directly to the root logger,
+    # which will catch events from all loggers going forward (propagation is on by
+    # default).
+    #
+    # For more info: https://docs.python.org/3/howto/logging.html#handlers
+    logging.getLogger().addHandler(rr.log.text.LoggingHandler())
+    logging.getLogger().setLevel(-1)
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser(description="Describe your example here!")
```

Comment:
Actually, I've seen that so many of our current python code has just this line copy-pasted mindlessly, and so we should anticipate that:

```suggestion
    parser = argparse.ArgumentParser(description="Example of using the Rerun visualizer")
```